src/components/bulk-selection-snackbar/CloseIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+function CloseIcon() {
```

Comment:
There is no other Close Icon already? If not abstract this one and make it general.